testnet/stacks-node/src/neon_node.rs

```diff
@@ -650,18 +650,22 @@ impl InitializedNeonNode {
             .expect("Unexpected BurnDB error fetching block commits");
         for op in block_commits.into_iter() {
             if op.txid == block_snapshot.winning_block_txid {
```

Comment:
These changes will log all of the block commits that ever get submitted (whether you are a miner or not), so could get pretty chatty. Can this be a debug log rather than info?